quizblock/models.py

```diff
@@ -225,6 +225,9 @@ class Question(models.Model):
         ))
     explanation = models.TextField(blank=True)
     intro_text = models.TextField(blank=True)
+    css_extra = models.TextField(
```

Comment:
New fields should be added to the model's as_dict method, and to Quiz's import_from_dict.